README.md

```diff
@@ -33,37 +33,16 @@ For cloud-based deployments, there are Docker images and Kubernetes configs:
 **[Detailed instructions for Kubernetes deployments](deploy/KUBERNETES.md)**  
 
 
-## Updating / Migrating an older xBrowse Instance
+## Updating a local installation from the v01 to the v02 hail/elasticsearch loading pipeline:
```

Comment:
I get why you added these instructions, but why did you delete the old, perfectly good instructions for upgrading other things?